clientd3d/map.c

```diff
@@ -526,7 +528,9 @@ void MapDrawAnnotations( HDC hdc, MapAnnotation *annotations, int x, int y, floa
 		new_x = x + (int) (annotations[i].x * scaleToUse);
 		new_y =	y + (int) (annotations[i].y * scaleToUse);
 
-		radius = max(MAP_ANNOTATION_MIN_SIZE / 2, (int) (MAP_ANNOTATION_SIZE * scaleToUse / 2));
+      // Scale annotation, capping it between the minimum and maximum limits
+      capped_scaled_size = min(MAP_ANNOTATION_SIZE * scaleToUse, MAP_ANNOTATION_MAX_SIZE);
+      radius = max(MAP_ANNOTATION_MIN_SIZE, capped_scaled_size) / 2;
```

Comment:
Seems like there might be an inconsistency here.  ANNOTATION_MAP_SIZE refers to the whole size (diameter) of the object, but MAP_OBJECT_MAX_SIZE refers to the half size (radius) in line 387 above?

Reply:
Yes, it does seem like an inconsistency. I tried to keep things largely as they were and it would seem like renaming `MAP_OBJECT_MAX_SIZE` to `MAP_OBJECT_MAX_RADIUS` could clear this up. Would you like me to take a look at reworking annotations to be based on radius or vice versa? (Assuming this is what you're commenting on)

Reply:
Yes, I think we should.  Either one is fine as long as they're all consistent.

Reply:
👍🏻

I felt it made the most sense to switch annotations over to radius seeing as that the position code is all based around having the radius.